src/test/java/uk/nhs/ctp/service/TriageServiceTest.java

```diff
@@ -140,4 +122,34 @@ public void testQuestionnaireRequestNotMadeWhenOnlyResultIsPresent() throws FHIR
     verify(cdssService, times(0)).getQuestionnaire(anyLong(), anyString());
   }
 
+  @Test
+  public void processTriageRequest_withCarePlanIds_willCompleteCarePlans() throws Exception {
+    var carePlanIds = new String[]{"id1", "id2"};
+    when(mockCdssRequestDTO.getCarePlanIds()).thenReturn(carePlanIds);
```

Comment:
why mock a request dto? wouldn't it be easier just to pass one in with the care plan ids set (plus whatever else is the minimum)?

Reply:
Now that you mention it, there's no reason to - I initially wanted to have a minimal impact on existing code, but I do agree they need improving - will update!